proposals/nnnn-structured-concurrency.md

```diff
@@ -548,13 +548,15 @@ extension Task {
   @discardableResult
   static func runDetached<T>(
     priority: Priority = .default,
+    startingOn executor: ExecutorRef? = nil,
```

Comment:
Great, I think that's necessary 👍 Naming bikeshed I was hoping for `runDetached(on: .ioExecutor)` etc

Reply:
I was concerned that `on:` might imply a stronger attachment to the executor than there really is, since many tasks are likely to do something actor-bound and end up hopping to a different executor at some point in their lives.

Reply:
That's true... we do want to strongly guarantee we always start on the passed in executor _if_ there is one passed tho I guess, and if there is _not_ we could potentially see this as optimization chance to avoid a hop if it isnt necessary? (see my example below where group.add immediately calls an actor func)